formulas/wick-base/explorers/os

```diff
@@ -62,6 +62,19 @@ if [[ -f /etc/arch-release ]]; then
     exit
 fi
 
+# Check os-release files for ID
+if [[ -f /etc/os-release ]]; then
+    . /etc/os-release
+elif [[ -f /usr/lib/os-release ]]; then
+    . /usr/lib/os-release
+fi
+# ID is lowercase distro name and defaults to "linux" if missing
```

Comment:
Newline above this comment.

Does the $ID always match what the normal output is from wick?  I would prefer that we run this through a case statement to only allow specific values.